src/node_crypto.cc

```diff
@@ -5533,9 +5533,9 @@ void PBKDF2Request::After() {
 
 void PBKDF2Request::After(uv_work_t* work_req, int status) {
   CHECK_EQ(status, 0);
-  PBKDF2Request* req = ContainerOf(&PBKDF2Request::work_req_, work_req);
+  std::unique_ptr<PBKDF2Request> req(
+    ContainerOf(&PBKDF2Request::work_req_, work_req));
```

Comment:
4 space indent.  (Line continuation.)

Reply:
Thanks! I think I have a few of these wrong. Let me check my other PRs.